edit-post/assets/stylesheets/main.scss

```diff
@@ -102,11 +102,46 @@ body.gutenberg-editor-page {
 		min-height: calc( 100vh - #{ $admin-bar-height-big } );
 	}
 
-	// The WP header height changes at this breakpoint
+	// The WP header height changes at this breakpoint.
 	@include break-medium {
 		min-height: calc( 100vh - #{ $admin-bar-height } );
 	}
 
+	.components-navigate-regions {
+		height: 100%;
+	}
+}
+
+/**
+ * Vanilla editor styles
+ */
+
+.gutenberg__editor {
+
+	// Gutenberg specific elements
+	.editor-default-block-appender__content,
+	.components-placeholder {
+		margin-top: $default-block-margin;
+		margin-bottom: $default-block-margin;
+	}
+
+	// Core blocks
+	.wp-block-button,
+	.wp-block-code,
+	.wp-block-categories,
+	.wp-block-cover-image,
+	.wp-block-embed {
+		margin-top: $default-block-margin;
+		margin-bottom: $default-block-margin;
+	}
+
+	p,
```

Comment:
Should these rules be duplicated across each and every block in the style.scss file for each, or is it fine to keep these in a global vanilla theme like these?